api/auth/jwtUtils.test.js

```diff
@@ -6,6 +6,8 @@ const jwt = require('jsonwebtoken');
 const sandbox = sinon.createSandbox();
 const mockVerifier = sandbox.stub();
 
+const JWT_SECRET = process.env.JWT_SECRET || 'SOME_SECRET_VALUE_SHHHHH'
```

Comment:
If JWT_SECRET isn't set from the env and defaults to that string it will be a sort of silent failure/security concern, wont it? Mostly just curious. I've always seen fallback values and never really question it but I wonder if not having a fallback and letting the system choke on startup would be preferred.